src/components/Card/Card.jsx

```diff
@@ -1,29 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
+
+import { FavoriteProcess } from '../../utils/favoriteProcess';
+import StarIcon from '../../ui/icons/Star';
+import { useSession } from '../../state/Provider';
+
 import {
   Container,
   ImageFit,
   GradientContainer,
   Title,
   Description,
   TagChannel,
+  FavStar,
 } from './Card.styled';
 
-const Card = ({ id, title, description, imgUrl, channelTitle, isFluid }) => {
+const Card = ({
+  id,
+  title,
+  description,
+  imgUrl,
+  channelTitle,
+  isFluid,
+  canFavorite,
+  linkBase = 'videos',
+}) => {
   const history = useHistory();
+  const [isShown, setIsShown] = useState(false);
```

Comment:
It might be a good idea to give a more meaningful name to this variable like this:

```suggestion
  const [isFavButtonShown, setIsFavButtonShown] = useState(false);
```

Reply:
Actually it is a good idea, I'll do a better variable naming, thank you José :D